src/resolve/resolvectl.c

```diff
@@ -2246,7 +2261,7 @@ static int verb_nta(int argc, char **argv, void *userdata) {
         if (r < 0)
                 return bus_log_create_error(r);
 
-        r = sd_bus_message_append_strv(req, argv + 2);
+        r = sd_bus_message_append_strv(req, clear ? empty_strv : argv + 2);
```

Comment:
I think assigning `NULL` is OK.

Reply:
Used STRV_MAKE_EMPTY here.

Reply:
Passing `NULL` and `STRV_MAKE_EMPTY` to `sd_bus_message_append_strv()` seem equivalent:
https://github.com/systemd/systemd/blob/385b2eb262a99373f09d01b7f5571dd71a14dc98/src/libsystemd/sd-bus/bus-message.c#L2762-L2772
Of course, this is OK, but it seems that we usually use `NULL` instead, e.g.,
https://github.com/systemd/systemd/blob/385b2eb262a99373f09d01b7f5571dd71a14dc98/src/machine/machinectl.c#L1544

Reply:
Ah indeed, didn't know STRV_FOREACH was smart enough to treat NULL as an empty strv... Thanks!